tox.ini

```diff
@@ -1,7 +1,8 @@
 [tox]
 minversion = 2.4
+envlist = py{2.7,3.6}
 
 [testenv]
+changedir = test
```

Comment:
Do the tests work without this change (keeping usedevelop)? I've found usedevelop is necessary for some packages, and I'd rather diverge as little as possible from the scaffolding that other projects need.